tests/ha/wait_others_upgraded.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Wait for all nodes which are allowed to upgrade after.
+# Maintainer: Christian Lanig <clanig@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use hacluster;
+use lockapi;
+
+sub run {
+
+    my $cluster_name = get_cluster_name;
+    if (is_node(1)) {
+        record_info("Upgrade done", "Node 1 successfully upgraded");
+        barrier_wait("NODE_UPGRADED_${cluster_name}_NODE1");
+        record_info("Waiting node 2", "Node 2 is upgrading");
+        barrier_wait("NODE_UPGRADED_${cluster_name}_NODE2");
+    }
+    barrier_wait("NODE_UPGRADED_${cluster_name}_NODE2") if is_node(2);
```

Comment:
L27 and L29 can be the same line (removing L27, and removing `if is_node(2)` from L29). And it would help to eventually cover 3+ cluster node scenarios.

Reply:
Done, VR updated for 15-SP2 QAM. Thx